main/zoe/api/contract-support/price-authority.md

```diff
@@ -0,0 +1,141 @@
+# PriceAuthority Methods
```

Comment:
Just this morning I merged a PR that adds a bunch of new functionality that needs a separate discussion. The new methods are `mutableQuoteWhenLT,  mutableQuoteWhenLTE, mutableQuoteWhenGT`, and `mutableQuoteWhenGTE`.  The mutableQuoteWhen methods return a `mutableQuote` object with these methods: `cancel(e),   getPromise()`, and `updateLevel(newAmountIn, newAmountOutLimit)`.

`getPromise()` returns the same promise that you get from the `quoteWhen...()` methods, but additionally, you can call `updateLevel()` to change the trigger levels without requiring a second promise. This is designed to save effort and production of excessive objects (promises) for both the client of the priceAuthority and for the system.  Let me know if you have questions about the usage. `cancel()` causes the promise to `reject` with the message.

Reply:
I've made a new issue for this and will start work on it, but I'd just as soon go ahead and merge this in to get ratio math in the system.